wagtail/project_template/home/templates/home/welcome_page.html

```diff
@@ -0,0 +1,380 @@
+                </g>
+              </g>
+            </g>
+            <path fill="#9FD6E4" d="M75.325 68.613h.002-.002"/>
+            <path fill="#C7462F" d="M95.425 69.054h-.174.175c15.502 0 30.138-.877 43.064-2.432 2.372-5.235 3.81-10.982 4.097-17.027-.288 6.045-1.725 11.792-4.098 17.027-12.927 1.555-27.563 2.432-43.065 2.432m-.23 0h-.114.114m-.198 0h-.085.086m-.18 0h-.07.07m-.192 0h-.034.034m-.178 0h-.008.008"/>
+            <path fill="#1D2533" d="M137.33 26.445l-.09-.18.09.18m-.095-.19l-.036-.072c.01.023.024.047.035.072m-.06-.12l-.015-.03.016.03"/>
+            <path fill="#FFFFFE" d="M31.976 65.48c0 1.978-3.635 3.58-8.118 3.58-4.484 0-8.118-1.602-8.118-3.58 0-1.977 3.634-3.58 8.118-3.58 4.483 0 8.118 1.603 8.118 3.58M170.26 65.48c0 1.978-3.634 3.58-8.118 3.58s-8.118-1.602-8.118-3.58c0-1.977 3.634-3.58 8.118-3.58s8.118 1.603 8.118 3.58M58.957 76.666c0 1.977-3.634 3.58-8.117 3.58-4.483 0-8.118-1.603-8.118-3.58 0-1.976 3.635-3.58 8.118-3.58 4.483 0 8.117 1.604 8.117 3.58M145.525 76.666c0 1.977-3.634 3.58-8.117 3.58-4.484 0-8.118-1.603-8.118-3.58 0-1.976 3.634-3.58 8.118-3.58 4.483 0 8.117 1.604 8.117 3.58M103.93 80.022c0 1.977-3.636 3.58-8.12 3.58-4.483 0-8.118-1.603-8.118-3.58 0-1.976 3.635-3.58 8.12-3.58 4.482 0 8.117 1.604 8.117 3.58"/>
+          </g>
+        </g>
+      </g>
+    </svg>
+  </div>
+  <br>
+  <h2>{% templatetag openblock %} trans "Welcome to your new Wagtail site!" {% templatetag closeblock %}</h2>
+  <p>{% templatetag openblock %} trans 'Please feel free to <a href="https://wagtail.io/slack/" target="_blank">join our community on Slack</a>, or get started with one of the links below.' {% templatetag closeblock %}</p>
+</main>
+<footer class="u-clearfix">
+  <a href="http://docs.wagtail.io/" target="_blank" rel="noopener">
```

Comment:
Should be `rel="nofollower noopener"` for browsers that do not support noopener (IE11, Edge).

Reply:
Please use an HTTPS link instead.

Reply:
Fixed.